test/test_playbooks/tasks/realm.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: "Ensure realm '{{ realm_name }}' is {{ realm_state }}"
+  foreman_realm:
```

Comment:
Put some default variables here so that if I don't pass realm_name and realm_state, it'll still do something.

Reply:
@akofink Thanks.  I'll definitely do that. Thanks for the suggestion.

Reply:
@akofink So it looks like we can define variables in the server_vars.yml file.  We have a template that is available under test/test_playbooks.  In my test I added that to the vars to the realm.yml but they can be added to the server_vars.yml.  Is that the intent? The tests don't have a place to add group_vars etc.  Let me know if I am missing something.

Reply:
@claudiol Like [this](https://github.com/theforeman/foreman-ansible-modules/blob/master/test/test_playbooks/tasks/product.yml#L4-L6).